SimDataFormats/Track/src/classes_def.xml

```diff
@@ -13,4 +13,11 @@
   <class name="edm::RefProd<std::vector<SimTrack> >"/>
   <class name="edm::Ref<std::vector<SimTrack>,SimTrack,edm::refhelper::FindUsingAdvance<std::vector<SimTrack>,SimTrack> >"/>
   <class name="edm::RefVector<std::vector<SimTrack>,SimTrack,edm::refhelper::FindUsingAdvance<std::vector<SimTrack>,SimTrack> >"/>
+		<class name="UniqueSimTrackId"/>
```

Comment:
Do we need space like this here?

Reply:
I would fix indentation and restart the test before making other checks